web/assets/js/types/types.ts

```diff
@@ -2,6 +2,7 @@ export type Colors = 'w' | 'b';
 export type GameoverType =
   | 'checkmate'
   | 'timeout'
+  | 'noMove'
```

Comment:
?

Reply:
I needed two type, in ne for timeout (no more time) that calls for a winner. And another for the 30s rule that calls for a abandon.

Reply:
Quick and dirty.